GooglePay/src/test/java/com/braintreepayments/api/googlepay/GooglePayCardNonceUnitTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.braintreepayments.api.googlepay
+
+import android.os.Parcel
+import com.braintreepayments.api.core.PostalAddress
+import com.braintreepayments.api.sharedutils.Json
+import com.braintreepayments.api.testutils.Assertions.assertBinDataEqual
+import com.braintreepayments.api.testutils.Fixtures
+import kotlinx.parcelize.parcelableCreator
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@RunWith(RobolectricTestRunner::class)
+class GooglePayCardNonceUnitTest {
+
+    private fun getPostalAddressObjectFromJson(address: JSONObject): PostalAddress {
+        val result = PostalAddress().apply {
+            recipientName = Json.optString(address, "name", "")
+            streetAddress = Json.optString(address, "address1", "")
+            extendedAddress = listOf(
+                Json.optString(address, "address2", ""),
+                Json.optString(address, "address3", "")
+            ).joinToString(separator = "\n").trim()
+            locality = Json.optString(address, "locality", "")
+            region = Json.optString(address, "administrativeArea", "")
+            countryCodeAlpha2 = Json.optString(address, "countryCode", "")
+            postalCode = Json.optString(address, "postalCode", "")
+        }
+        return result
+    }
+
+    private fun assertPostalAddress(expected: PostalAddress, actual: PostalAddress) {
+        assertEquals(expected.toString(), actual.toString())
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun `parses JSON object and creates GooglePayCardNonce`() {
+        val response = Fixtures.PAYMENT_METHODS_GOOGLE_PAY_CARD_RESPONSE
+        val billing = JSONObject(response)
+            .getJSONObject("paymentMethodData")
+            .getJSONObject("info")
+            .getJSONObject("billingAddress")
+        val shipping = JSONObject(response).getJSONObject("shippingAddress")
+
+        val billingPostalAddress = getPostalAddressObjectFromJson(billing)
+        val shippingPostalAddress = getPostalAddressObjectFromJson(shipping)
+
+        val googlePayCardNonce = GooglePayCardNonce.fromJSON(JSONObject(response)) as GooglePayCardNonce
+
+        assertEquals("fake-google-pay-nonce", googlePayCardNonce.string)
+        assertEquals("Visa", googlePayCardNonce.cardType)
+        assertEquals("123456", googlePayCardNonce.bin)
+        assertEquals("11", googlePayCardNonce.lastTwo)
+        assertEquals("1234", googlePayCardNonce.lastFour)
+        assertEquals("android-user@example.com", googlePayCardNonce.email)
+        assertPostalAddress(billingPostalAddress, googlePayCardNonce.billingAddress)
+        assertPostalAddress(shippingPostalAddress, googlePayCardNonce.shippingAddress)
+        assertTrue { googlePayCardNonce.isNetworkTokenized }
```

Comment:
We can invoke the `assertTrue()` function that takes in a boolean value

```suggestion
        assertTrue(googlePayCardNonce.isNetworkTokenized)
```